test_unstructured_ingest/test-ingest-google-cloud.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e
+
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+cd "$SCRIPT_DIR"/.. || exit 1
+
+if [[ "$(find test_unstructured_ingest/expected-structured-output/google-cloud-storage/ -type f -size +2 | wc -l)" -ne 5 ]]; then
+    echo "The test fixtures in test_unstructured_ingest/expected-structured-output/ look suspicious. At least one of the files is too small."
+    echo "Did you overwrite test fixtures with bad outputs?"
+    exit 1
+fi
+
+PYTHONPATH=. ./unstructured/ingest/main.py \
+    --metadata-exclude filename,file_directory,metadata.data_source.date_processed \
+    --remote-url gs://unstructured_public/ \
```

Comment:
let's use this test folder: gs://utic-test-ingest-fixtures
it has two folders `public` and `private`, where private will require that the auth token is used. hopefully can leverage what [was done in the google drive test](https://github.com/Unstructured-IO/unstructured/blob/main/test_unstructured_ingest/test-ingest-google-drive.sh#L8-L27) that I just added?

Reply:
Worked. But had to `cat` the cred info into the tempfile instead of `echo`

Reply:
huh, weird. yea, as long as it works locally and in CI, that works for me.

Reply:
looking at CI, it doesn't look like linux is happy about the `cat` change. what's weird is that the way I had it before worked for me locally, maybe worth double checking?:

`cat: '***'$'\n''  ***'$'\n''  ***'$'\n''  ***'$'\n''  ***'$'\n''  ***'$'\n''  ***'$'\n''  ***'$'\n''  ***'$'\n''  ***'$'\n''  ***'$'\n''  ***'$'\n''***'$'\n': File name too long`

https://github.com/Unstructured-IO/unstructured/actions/runs/5330890729/jobs/9658103678

Reply:
ah, right, I see the confusion here. (should have noticed before). I think this wasn't working for you locally because you were setting GCP_INGEST_SERVICE_KEY to a filepath. GCP_INGEST_SERVICE_KEY is the actual key itself. This should works with echo on any platform.